Regression/Checksum/update_benchmarks_from_azure_output.py

```diff
@@ -0,0 +1,48 @@
+import re
+import sys
+
+azure_output_filename = sys.argv[1]
+
+first_char = 29 # first non prefix character
+test_name_str = "working on test:"
+test_name_first_char = 50 # start of test name
+test_name_last_char = -5 # end of test name
```

Comment:
I am a bit puzzled by some of these numbers. Looking at the raw logs, there seems to be usually an initial alphanumeric string (containing info like date, time, etc.) such as `2000-01-01T00:00:00.1234567A` (28 characters plus a trailing white space, so 29 characters in total). Then I seem to see a few weird characters such as a symbol plus `[1m` (another 4 characters), before the string `working on test:` starts. However, these additional 4 characters seem not to be included in your counting, when you set `first_char = 29` and `test_name_first_char = 50`. I see these additional characters both when I inspect that raw log on my browser and when I save to a text file locally.

Reply:
I think there's two answers to that one:
- The short one: actually I didn't really try to understand those numbers and just found empirically what worked in my python script. :smile:
- The long one: these weird characters are here for formatting (in this case, making the text bold when viewing the output). I'm not exactly sure where your count is off by 4 characters compared to mine, but maybe it's because `'\x1b'` is read as a single character in a python string? In any case, I'm showing below the output of what I get when reading one of these lines with python, so that you can see why I chose 50 here.

![image](https://user-images.githubusercontent.com/49716072/135836164-ab635584-779d-4b57-8202-8f3938982c95.png)